buildchain/buildchain/vagrant.py

```diff
@@ -40,16 +42,53 @@ def mkdir_dot_vagrant() -> None:
         'uptodate': [True],
     }
 
+def task_vagrantupnoprov() -> types.TaskDict:
```

Comment:
Does the user wants to use this?

If not and the only purpose is to use it as `task_dep`, this should be a private task (`task__vagrantupnoprov`) to avoid polluting the users with useless tasks when they run `doit list`.

Same remark for `task_vagrantsnapshot`.